lms/static/sass/shared/_fields.scss

```diff
@@ -12,7 +12,7 @@
         border-radius: 3px;
 
         span {
-            color: $gray-l1;
+            color: $gray-d1;
```

Comment:
Also, was ``$gray`` not a viable option (keeping in mind the original color of the UI)?